### PR TITLE
CB-9121 Add support for build configuration to be specified using the CLI

### PR DESCRIPTION
**Pull request by nikhilkh**

### CODE REVIEW

File: doc/build.txt [hunk 1]
```diff
@@ -18,6 +19,9 @@ all/the specified platforms.
     --emulator ......................... Deploy to an emulator
     --target ........................... Deploy to a specific target
 
+    --buildConfig....................... Use the specified build configuration
```
Comment (omefire) [inline doc/build.txt]: It'd be great if you could document an example of how to use this flag in both build.txt and run.txt

File: src/cli.js [hunk 2]
```diff
@@ -226,7 +227,9 @@ function cli(inputArgs) {
                 downstreamArgs.push('--' + flag);
             }
         });
-
+        if (args.buildConfig) {
+            downstreamArgs.push('--buildConfig=' + args.buildConfig);
```
Comment (omefire) [inline src/cli.js]: don't we need a corresponding PR for cordova-lib ?

Comment (omefire) [inline src/cli.js]: ok, never mind. it seems like cordova-lib just passes the --buildConfig args to the underlying platform, which does the bulk of the work : https://github.com/apache/cordova-android/pull/164

**Comment by omefire** [comment]
LGTM!

File: doc/build.txt (hunk 1 above)
Comment (TimBarham) [inline doc/build.txt]: :+1: ... At the very least, it should have (on line 9) `--buildConfig=CONFIGFILE` or something like that.

**Comment by TimBarham** [comment]
:+1: apart from the one small thing.

File: doc/build.txt (hunk 1 above)
Comment (nikhilkh) [inline doc/build.txt]: Addressed both comments - added an example and fixed the grammar definition on line 9.

Comment (TimBarham) [inline doc/build.txt]: :shipit:!

**Pull request merged**